physics/mynnedmf_wrapper.F90

```diff
@@ -475,6 +475,32 @@ SUBROUTINE mynnedmf_wrapper_run(        &
                 qnifa(i,k) = qgrs_ice_aer_num_conc(i,k)
               enddo
             enddo
+          else if(mraerosol) then
+            FLAG_QI = .true.
+            FLAG_QNI= .true.
+            FLAG_QC = .true.
+            FLAG_QNC= .true.
+            FLAG_QNWFA= .false.
+            FLAG_QNIFA= .false.
+            p_qc = 2
+            p_qr = 0
+            p_qi = 2
+            p_qs = 0
+            p_qg = 0
+            p_qnc= 0
```

Comment:
I readily admit that I do not understand what is being attempted in the PBL scheme, but I have my guess.  I assume that variables marked with value of 2 are getting PBL tendencies for mixing.  As such, I think that means that mixing ratios of cloud water and cloud ice are getting turbulent mixing.  Yet why not number concentrations?

Reply:
Those look like a deprecated feature. I didn't see the code actually used it. Maybe just a placeholder.